README.md

```diff
@@ -1,49 +1,40 @@
-08. `sh adobe-rmsdk-build.sh`
-09. `(cd readium-sdk; sh MakeHeaders.sh Apple)` (parentheses included) to generate the headers for Readium.
-12. `open Simplified.xcodeproj`
-13. Build
-
-## Building OpenSSL and cURL
-
-In theory, following the instructions in "adobe-rmsdk/RMSDK_User_Manual(obj).pdf", you should be able to build OpenSSL (section 12.1) and cURL (section 12.3) since Adobe provides this package to their developers.
-The following are some smoother steps to achieving this:
-01. From `adobe-rmsdk/thirdparty/openssl` run `mkdir public; mv iOS-openssl/ public/ios/`
-02. Download openssl 1.0.1 (1.0.1u should suffice) to `adobe-rmsdk/thirdparty/openssl/public/ios/openssl-1.0.1u.tar.gz` (note that the build script is expecting a tarball; as of this writing the download can be found at: https://www.openssl.org/source/old/1.0.1/)
-03. Modify the `adobe-rmsdk/thirdparty/openssl/public/ios/build.sh` such that:
-    - `OPENSSL_VERSION` reflects the version you downloaded, in this case "1.0.1u"
-    - `SDK_VERSION` reflects the iOS SDK you're using (you can check what you have installed using `xcodebuild -showsdks`)
-    - `MIN_VERSION` is "9.0"
-04. From the directory of the build script (this will take a while): `bash ./build.sh`
-05. Download curl (7.64.1 is known to work) to `adobe-rmsdk/thirdparty/curl/ios-libCurl/curl-7.64.1.zip` (note that the build script expects a ZIP, not tarball; as of this writing this download can be found at: https://curl.haxx.se/download/)
-06. Replace `adobe-rmsdk/thirdparty/curl/ios/build.sh` with `build_curl.sh`, a fixed version of the build script
-07. Modify the build script similary to the openssl one:
-    - `CURL_VERSION` reflects the version you downloaded, in this case "7.64.1"
-    - `SDK_VERSION` reflects the iOS SDK you're using (you can check what you have installed using `xcodebuild -showsdks`)
-    - `MIN_VERSION` is "9.0"
-08. From the directory of the build script (this will take a while): `bash ./build.sh` (or whatever you named the copied script)
-
-Be sure to note the following:
-
-* You will need to verify and edit the "build.sh" scripts for both OpenSSL and cURL to reflect the correct version numbers and local directory names (lines 11–24).
-* You must add `--enable-ipv6` to line 80 of Adobe's "build.sh" script used for building cURL. This necessary both due to Apple's requirements for IPv6 support and because the library may not build against recent iOS SDKs otherwise.
-* cURL 7.57.0 is known _not_ to work and later versions are unlikely to work either. 7.50.0 is recommended.
+01. Contact project lead and ensure you have repo access to all required submodules, including private ones. Also request a copy of the Adobe RMSDK archive, which is currently not on Github, unzip it and place it in a place of your choice.
+02. Then run:
+```bash
+git clone git@github.com:NYPL-Simplified/Simplified-iOS.git
+git clone git@github.com:NYPL-Simplified/Certificates.git
```

Comment:
Just raising a concern over security by obscurity. While it's weak, it's still something.
It might be worth considering just referencing it by something like "the secrets repo"
And perhaps in the future we can make the installation location more arbitrary rather than making the relative pathing set in stone.

Reply:
IMO this is ok: while security by obscurity is a practice, a serious hacker would not be stopped by that in any way. They would still need to break github's security, which is really what defines our security. 

Besides, the android repos do something similar in publicly available repos to automate their builds too, e.g.:
https://github.com/NYPL-Simplified/Simplified-Android-SimplyE/blob/e9d9d1b9353e54b428e0cbb6301e54c0c324674c/.travis-pre.sh
https://github.com/NYPL-Simplified/Simplified-Android-Core/blob/b76381f60cfcb47c113dc3554509018897712641/.travis-pre.sh

@MalcolmMcFly thoughts?

Reply:
I'm ok either way here. If they can get past GitHub security, we don't stand a chance anyway. I think `Clone Certificate repo` would work too.